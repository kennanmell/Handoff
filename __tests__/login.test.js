import 'react-native';
import React from 'react';
import FacebookLoginPage from '../View/login';
import renderer from 'react-test-renderer';

// Ensures that the login page (FacebookLoginPage) renders correctly.
it('renders correctly', () => {
  const tree = renderer.create(
    	<FacebookLoginPage
    		onUserLogin={ () => {
    		}}
    		
    		onOrgLogin={ () => {
    		}}
    	/>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});