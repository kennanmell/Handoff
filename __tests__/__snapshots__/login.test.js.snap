exports[`test renders correctly 1`] = `
<View>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "fontSize": 20,
        "textAlign": "center",
      }
    }>
    Handoff Login
  </Text>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "#612e5f",
        "fontSize": 15,
        "fontWeight": "bold",
        "margin": 0,
        "textAlign": "left",
      }
    }>
    Name
  </Text>
  <TextInput
    onChangeText={[Function]}
    style={
      Object {
        "height": 40,
      }
    } />
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "#612e5f",
        "fontSize": 15,
        "fontWeight": "bold",
        "margin": 0,
        "textAlign": "left",
      }
    }>
    Password
  </Text>
  <TextInput
    onChangeText={[Function]}
    style={
      Object {
        "height": 40,
      }
    } />
  <View
    accessibilityComponentType={undefined}
    accessibilityLabel={undefined}
    accessibilityTraits={undefined}
    accessible={true}
    hitSlop={undefined}
    onLayout={undefined}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Array [
        Object {
          "backgroundColor": "transparent",
        },
        Object {
          "alignItems": "flex-end",
          "backgroundColor": "#642D64",
          "borderColor": "black",
          "borderWidth": 1,
          "flexDirection": "column",
          "justifyContent": "center",
          "marginBottom": 15,
          "marginLeft": 3,
          "marginTop": 10,
          "padding": 5,
        },
      ]
    }
    testID={undefined}>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#FFFFFF",
        }
      }>
      Login
    </Text>
  </View>
  <View
    accessibilityComponentType={undefined}
    accessibilityLabel={undefined}
    accessibilityTraits={undefined}
    accessible={true}
    hitSlop={undefined}
    onLayout={undefined}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Array [
        Object {
          "backgroundColor": "transparent",
        },
        Object {
          "alignItems": "flex-end",
          "backgroundColor": "#642D64",
          "borderColor": "black",
          "borderWidth": 1,
          "flexDirection": "column",
          "justifyContent": "center",
          "marginBottom": 15,
          "marginLeft": 3,
          "marginTop": 10,
          "padding": 5,
        },
      ]
    }
    testID={undefined}>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#FFFFFF",
        }
      }>
      New Organization
    </Text>
  </View>
  <View
    accessibilityComponentType={undefined}
    accessibilityLabel={undefined}
    accessibilityTraits={undefined}
    accessible={true}
    hitSlop={undefined}
    onLayout={undefined}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Array [
        Object {
          "backgroundColor": "transparent",
        },
        Object {
          "alignItems": "flex-end",
          "backgroundColor": "#642D64",
          "borderColor": "black",
          "borderWidth": 1,
          "flexDirection": "column",
          "justifyContent": "center",
          "marginBottom": 15,
          "marginLeft": 3,
          "marginTop": 10,
          "padding": 5,
        },
      ]
    }
    testID={undefined}>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#FFFFFF",
        }
      }>
      Donor Login
    </Text>
  </View>
  <View
    accessibilityComponentType={undefined}
    accessibilityLabel={undefined}
    accessibilityTraits={undefined}
    accessible={true}
    hitSlop={undefined}
    onLayout={undefined}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Array [
        Object {
          "backgroundColor": "transparent",
        },
        Object {
          "alignItems": "flex-end",
          "backgroundColor": "#642D64",
          "borderColor": "black",
          "borderWidth": 1,
          "flexDirection": "column",
          "justifyContent": "center",
          "marginBottom": 15,
          "marginLeft": 3,
          "marginTop": 10,
          "padding": 5,
        },
      ]
    }
    testID={undefined}>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#FFFFFF",
        }
      }>
      Subscriptions
    </Text>
  </View>
</View>
`;
